src/Component/spec/Metadata/RegistrySpec.php

```diff
@@ -11,11 +11,10 @@
 
 declare(strict_types=1);
 
-namespace spec\Sylius\Component\Resource\Metadata;
+namespace spec\Sylius\Resource\Metadata;
```

Comment:
no change here, the legacy file has been moved and the new one has been placed here.